ufpy/ustack.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+from typing import Any, Generic, TypeVar
+
+VT = TypeVar("VT")
+
+__all__ =  (
+    "UStack",
+)
+
+class UStack(Generic[VT]):
+    def __init__(self, __list: list[VT] = list()) -> UStack:
```

Comment:
```suggestion
    def __init__(self, __list: list[VT] = []):
```
init is not returning UStack :)